.eslintrc.js

```diff
@@ -32,10 +32,23 @@ module.exports = {
     "arrow-parens": ["error", "as-needed"],
     "prettier/prettier": ["error", { "singleQuote": true, "arrowParens": "avoid" }],
     "node/no-deprecated-api": ["warn"],
-    "header/header": [2, "block", [{
-        pattern: / /gm, 
-        template: ''
-    }]]
+    "header/header": [2, "block", [
+        "",
+        " * Copyright 2021 Splunk Inc.",
```

Comment:
Just FYI, Java does not include year into copyright: https://github.com/signalfx/splunk-otel-java/blob/main/gradle/spotless.license.java